feature-libs/my-account/organization/src/styles/_cost-centers.scss

```diff
@@ -0,0 +1,33 @@
+cx-cost-center-list {
+  > cx-split-view > cx-view > section > cx-table > table {
```

Comment:
Did we consider having lower CSS specificity? I believe having it such big one can make it harder to overwrite by customers.
(same for other `> ... > >`  chains)